doc/source/howtos/proxysql-v1.rst

```diff
@@ -0,0 +1,424 @@
+     --cluster-hostname=host_name       Hostname for connecting to the Percona XtraDB Cluster node
+     --cluster-app-username=user_name   Application username for connecting to the Percona XtraDB Cluster node
+     --cluster-app-password[=password]  Application password for connecting to the Percona XtraDB Cluster node
+     --without-cluster-app-user         Configure Percona XtraDB Cluster without application user
+     --monitor-username=user_name       Username for monitoring Percona XtraDB Cluster nodes through ProxySQL
+     --monitor-password[=password]      Password for monitoring Percona XtraDB Cluster nodes through ProxySQL
+     --without-check-monitor-user       Configure ProxySQL without checking/attempting to create monitor user
+     --enable, -e                       Auto-configure Percona XtraDB Cluster nodes into ProxySQL
+     --disable, -d                      Remove any Percona XtraDB Cluster configurations from ProxySQL
+     --node-check-interval=3000         Interval for monitoring node checker script (in milliseconds)
+     --mode=[loadbal|singlewrite]       ProxySQL read/write configuration mode, currently supporting: 'loadbal' and 'singlewrite' (the default) modes
+     --write-node=host_name:port        Writer node to accept write statments. This option is supported only when using --mode=singlewrite
+                                        Can accept comma delimited list with the first listed being the highest priority.
+     --include-slaves=host_name:port    Add specified slave node(s) to ProxySQL, these nodes will go into the reader hostgroup and will only be put into
+                                        the writer hostgroup if all cluster nodes are down.  Slaves must be read only.  Can accept comma delimited list.
+                                        If this is used make sure 'read_only=1' is in the slave's my.cnf
+     --adduser                          Adds the Percona XtraDB Cluster application user to the ProxySQL database
+     --syncusers                        Sync user accounts currently configured in MySQL to ProxySQL (deletes ProxySQL users not in MySQL)
+     --sync-multi-cluster-users         Sync user accounts currently configured in MySQL to ProxySQL (Don't delete ProxySQL users not in MySQL)
+     --version, -v                      Print version info
+   
+.. note:: Before using the ``proxysql-admin`` tool,
+   ensure that ProxySQL and |PXC| nodes you want to add are running. For
+   security purposes, please ensure to change the default user settings in
+   the ProxySQL configuration file.
+
+Preparing Configuration File
+----------------------------
+
+It is recommended to provide connection and authentication information
+in the ProxySQL configuration file (:file:`/etc/proxysql-admin.cnf`),
+instead of specifying it on the command line.
+
+By default, the configuration file contains the following:
+
+.. code-block:: text
+
```

Comment:
This cnf file is also old:
Here's the latest one

# proxysql admin interface credentials.
export PROXYSQL_DATADIR='/var/lib/proxysql'
export PROXYSQL_USERNAME='admin'
export PROXYSQL_PASSWORD='admin'
export PROXYSQL_HOSTNAME='localhost'
export PROXYSQL_PORT='6032'

# PXC admin credentials for connecting to pxc-cluster-node.
export CLUSTER_USERNAME='admin'
export CLUSTER_PASSWORD='admin'
export CLUSTER_HOSTNAME='localhost'
export CLUSTER_PORT='3306'

# proxysql monitoring user. proxysql admin script will create this user in pxc to monitor pxc-nodes.
export MONITOR_USERNAME='monitor'
export MONITOR_PASSWORD='monit0r'

# Application user to connect to pxc-node through proxysql
export CLUSTER_APP_USERNAME='proxysql_user'
export CLUSTER_APP_PASSWORD='passw0rd'

# ProxySQL read/write hostgroup 
export WRITE_HOSTGROUP_ID='10'
export READ_HOSTGROUP_ID='11'

# ProxySQL read/write configuration mode.
export MODE="singlewrite"

# Writer-is-reader configuration
export WRITER_IS_READER="ondemand"

# max_connections default (used only when INSERTing a new mysql_servers entry)
export MAX_CONNECTIONS="1000"